### DISCUSSION TITLE
The beatmap info doesn't fit all tags when there are many

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#24084] [Q&A]
I can't see all of the tags in the beatmap info of some beatmaps with many tags. Scrolling doesn't work and there's no "show all" button. I doubt that this is intended.

# Screenshot

![image](https://github.com/ppy/osu/assets/101510334/56bbe8de-db75-4071-a031-d11c8eb36f51)

# Beatmap

https://osu.ppy.sh/beatmapsets/1841846#osu/3783140

**Comment by N0Y0U2114** [comment]
Was about to make this same report. Still unaddressed. Commenting to push it back into view
